python: can't open file '/scratch/gpfs/sl4318/working_dir/Model': [Errno 2] No such file or directory
